balancer/xds/xds.go

```diff
@@ -479,6 +537,7 @@ func (x *xdsBalancer) startFallBackBalancer(c *xdsConfig) {
 	// balancer is registered or not.
 	builder := balancer.Get(c.FallBackPolicy.Name)
 	x.fallbackLB = builder.Build(x.cc, x.buildOpts)
+
 	if x.fallbackInitData != nil {
 		// TODO: uncomment when HandleBalancerConfig API is merged.
```

Comment:
What was this for?

Reply:
I have a feeling that this won't be as trivial as it sounds. It may change the way you handle  resolver updates in this PR.